### ISSUE TITLE
Flask themes fallback mode not activating with the latest code

### ISSUE THREAD

**Issue by whatnick** [opendatacube/datacube-explorer#169]
Latest code does not seem to be activating `flask-theme` fallback mode to find templates in the _templates directory. Thus no pages are rendering.

![image](https://user-images.githubusercontent.com/491396/88654203-61e06800-d110-11ea-979f-df3ca1eeba31.png)

**Comment by jeremyh** [comment]
Strange. They're rendering okay locally and in the tests. I've tried several variations of theme settings (including lack of settings) locally and it's working.

Can you post your `settings.env.py`?

I actually don't understand having this error message for that line: nothing is being iterated (except `context`, but it can't be None by Python's rules). Is there a deeper layer in the stacktrace that the message is coming from?

**Comment by jeremyh** [comment]
The line below it is actually more interesting: TemplateNotFound. My suspicion was that it's something environment-related, that the theme folder is not accessible. But I can't reproduce that exact error message just from locally deleting/moving templates or theme folders.

It's also interesting because it's mentioning `_themes/dea/overview.thml` ... but none of Explorer's theme code uses a folder called `_themes`. The string doesn't exist in this repository.

(... and this release didn't change anything with the theme setup, so it's odd that we're seeing this now)

**Comment by whatnick** [comment]
Here is the deployment config hopefully useful:

```
      CUBEDASH_THEME='dea'
      CUBEDASH_DEFAULT_PRODUCTS = ('aster_false_colour','ls5_fc_albers','ls7_fc_albers',)

      # Which field should we use when grouping products in the top menu?
      CUBEDASH_PRODUCT_GROUP_BY_FIELD = 'product_type'

      # Ungrouped products will be grouped together in this size.
      CUBEDASH_PRODUCT_GROUP_SIZE = 5

      # Maximum search results
      CUBEDASH_HARD_SEARCH_LIMIT = 100

      # Maximum number of source/derived datasets to show
      CUBEDASH_PROVENANCE_DISPLAY_LIMIT = 30

      # Customise '/stac' endpoint information
      STAC_ENDPOINT_ID = 'AWS_EKS_Dev_Explorer'
      STAC_ENDPOINT_TITLE = 'AWS eks dev explorer'
      STAC_DEFAULT_PAGE_SIZE = 20
      STAC_PAGE_SIZE_LIMIT = 1000

      CUBEDASH_DEFAULT_SCHEMA = 'cubedash'

      # Sentry access
      SENTRY_CONFIG = {'dsn':'https://magic@magic/magic'}
```

**Comment by jeremyh** [comment]
Ah, Loki! I can see the full stacktrace there now. 

This is a template render error because it's expecting data that's generated with the new version of `cubedash-gen`, but it's coming back null -- I believe this is fixed in #167, which makes that macro code more forgiving.

**Comment by jeremyh** [comment]
Reproduced in #170

**Comment by jeremyh** [comment]
Closing. It was incompatibility with the older schema. Fixed in #170.
